ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -389,6 +401,11 @@ public Builder propagationManagerGetBlockTimeoutMillis(
       return this;
     }
 
+    public Builder setNearHeadCheckpointSyncEnabled(final boolean nearHeadCheckpointSyncEnabled) {
```

Comment:
Builder pattern convention is to omit 'set'

```suggestion
    public Builder nearHeadCheckpointSyncEnabled(final boolean nearHeadCheckpointSyncEnabled) {
```